mps_youtube/helptext.py

```diff
@@ -79,6 +79,8 @@ def helptext():
     {2}dvupl <username>{1} - download user's YouTube playlists (video).
     {2}dlurl <url or id>{1} download a YouTube video by url or video id.
     {2}playurl <url or id>{1} play a YouTube video by url or id.
+    {2}browserplay <url or id>{1} play a YouTube video by url or id in browser.
```

Comment:
I mean that it should do something like @bendem said, if that isn't clear.

That seems to be where this is particularly useful. For an url, this is fairly useless, since you can just enter it into the browser directly. Handling an id is mildly more useful, but usually you get one from looking at the url anyway. Taking a number for a search result, playlist item, etc. is where it is useful, and integrates with mpsyt's functionality.